src/styles/_variables.scss

```diff
@@ -1,8 +1,10 @@
 $borderColor: #000;
 $buttonLinkColor: #fff;
 $hoverColor: #5a3149;
-$mainTextColor: #000;
+$mainTextColor: #fff;
 $outlineColor: #000;
 $mainGreenColor: #315a42;
 $mainBrownColor: #5a5731;
-$mainBlueColor: #31355a;
\ No newline at end of file
+$mainBlueColor: #31355a;
```

Comment:
Something to think about - Naming your variables things like "green" and "brown" means if you ever change the theme you'll have to either deal with the variable being named incorrectly or go through and change it across everything. Personally I like to do something like this: 

$brownOne: #5a5731;
$backgroundPrimary: $brownOne;